Apps/Common/src/Models/Address.cs

```diff
@@ -0,0 +1,60 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.Common.Models
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Represents an address.
+    /// </summary>
+    public class Address
+    {
+        /// <summary>
+        /// Gets or sets the street lines.
+        /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "Team Decision")]
+        public IList<string> StreetLines { get; set; } = new List<string>();
```

Comment:
We need a constructor for this since we are not suppressing CA2227

Reply:
Fixed